### PR TITLE
punto8

### PR DESCRIPTION
**Pull request by jeffrey20122021**

### CODE REVIEW

File: buy-shirt.cy.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        cy.get('#cgv')
+        cy.get('.cart_navigation > .button > span')
+        cy.get('.bankwire')
+        cy.get('#cart_navigation > .button > span')
+        cy.get("#center_column > div > p > strong").should(
+            "have.text",
+            "Your order on My Store is complete."
+        );
+    });
+});
```
Comment (veronicatofino) [inline buy-shirt.cy.ts:24]: EOF

File: buy-shirt.cy.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        cy.visit("http://automationpractice.com/");
+        cy.get("#block_top_menu > ul > li:nth-child(3) > a").click();
+        cy.get(
+            "#center_column a.button.ajax_add_to_cart_button.btn.btn-default"
+        ).click();
+        cy.get("[style*='display: block;'] .button-container > a").click();
+        cy.get(".cart_navigation span").click();
+        cy.get("#email").type("aperdomobo@gmail.com");
+        cy.get("#passwd").type("WorkshopProtractor");
+        cy.get("").type("WorkshopProtractor");
```
Comment (veronicatofino) [inline buy-shirt.cy.ts:12]: Is this step necessary?

File: buy-shirt.cy.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
+describe("Buy a t-shirt", () => {
```
Comment (veronicatofino) [inline buy-shirt.cy.ts:1]: Please create this test in cypress/e2e folder

File: buy-shirt.cy.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        cy.get("#block_top_menu > ul > li:nth-child(3) > a").click();
+        cy.get(
+            "#center_column a.button.ajax_add_to_cart_button.btn.btn-default"
+        ).click();
+        cy.get("[style*='display: block;'] .button-container > a").click();
+        cy.get(".cart_navigation span").click();
+        cy.get("#email").type("aperdomobo@gmail.com");
+        cy.get("#passwd").type("WorkshopProtractor");
+        cy.get("").type("WorkshopProtractor");
+        cy.get('.login');
```
Comment (veronicatofino) [inline buy-shirt.cy.ts:13]:
```suggestion
        cy.get('.login').click();
```

Comment (veronicatofino) [reply buy-shirt.cy.ts:13]: .login is not the correct selector for that button

**Review by veronicatofino — COMMENTED** [review]
Add a snapshot with the test in green locally

File: buy-shirt.cy.ts (hunk 2 above)
Comment (jeffrey20122021) [reply buy-shirt.cy.ts]: That steps are in the instructions of the exercise. @veronicatofino

**Comment by jeffrey20122021** [comment]
> Add a snapshot with the test in green locally

in local appear red @veronicatofino

File: buy-shirt.cy.ts (hunk 3 above)
Comment (jeffrey20122021) [reply buy-shirt.cy.ts]: done @veronicatofino

File: buy-shirt.cy.ts (hunk 1 above)
Comment (jeffrey20122021) [reply buy-shirt.cy.ts]: @veronicatofino done

File: cypress/e2e/buy-shirt.cy.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+      cy.visit("http://automationpractice.com/");
+      cy.get("#block_top_menu > ul > li:nth-child(3) > a").click();
+      cy.get(
+          "#center_column a.button.ajax_add_to_cart_button.btn.btn-default",
+      ).click();
+      cy.get("[style*='display: block;'] .button-container > a").click();
+      cy.get(".cart_navigation span").click();
+      cy.get("#email").type("aperdomobo@gmail.com");
+      cy.get("#passwd").type("WorkshopProtractor");
+      cy.get("").type("WorkshopProtractor");
```
Comment (holgiosalos) [inline cypress/e2e/buy-shirt.cy.ts:12]: this is not required, you are already typing the password in the previous line, the next should be the click in the sign in button

**Review by holgiosalos — COMMENTED** [review]
please run the lint:fix commad to fix the linter issues

File: cypress/e2e/buy-shirt.cy.ts (hunk 5 above)
Comment (jeffrey20122021) [reply cypress/e2e/buy-shirt.cy.ts]: done @holgiosalos

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**